docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup.md

```diff
@@ -0,0 +1,186 @@
+
+1. Create the following tables/indexes on Primary DB
+
+    ```sql
+    create database ybdb;
+    \c ybdb
+    create table ordering_test (id int);
+    create table customers (customer_id UUID, company_name text, customer_state text, primary key (customer_id));
+
+    create table orders(customer_id uuid, order_id int primary key, order_details jsonb, product_name text,product_code text, order_qty int, constraint fk_customer foreign key(customer_id) references customers(customer_id));
+    create index order_covering_index on orders (customer_id, order_id) include( order_qty); 
+    create table if not exists account_balance (id uuid, name text, salary int, primary key (id)); 
+
+    create index account_balance_secondary_index on account_balance(name);
+
+    create table if not exists account_sum ( sum int );
+    ```
+
+1. On Standby, create a fresh database with the same name (with and without data).
+
+    During initial replication setup, You are not required to create objects on standby cluster. xCluster Bootstrap auto-creates tables/objects and restores data on Standby from Primary cluster.
+
+1. Enable PITR using YBA UI on both Primary and Standby universes
+Backups > point-in-time-recovery.
+
+1. Set up xCluster Replication from PRIMARY to STANDBY universe using YBA UI.
+
+    If there is data in the databases then YBA will walk you through the backup & restore process.
+
+    **Note**: If there are any connections open against the target (standby) databases and replication is being set up for a Primary database that already has some data, then the set up replication with bootstrap flow will fail with an expected error. This is because setting up replication requires backing up the primary database and restoring the backup to the target database after cleaning up any pre-existing data on the target side. Please close any connections to the target(standby) database and retry the setup replication operation.
+
+    ```output
+    Error ERROR:  database "database_name" is being accessed by other users
+    DETAIL:  There is 1 other session using the database.
+    ```
+
+    **Note**: When you add a new database to an existing replication stream:
+
+    - you need to create the same objects on standby. If Source and Target objects do not match then DB will not be allowed to be added to replication.
+    - If the WALs are GC-ed from Primary then the DB will need to be bootstrapped. Bootstrap process is handled by the YBA. Only single DB is bootstrapped and not all the databases involved in existing Replication Stream.
+
+1. On Standby, verify `xcluster_safe_time` is progressing. This is the indicator of data flow from Primary to Standby.
+
+    (To know when to read from standby as of, YugabyteDB maintains an analog of safe time called xCluster safe time. This is the latest time it is currently safe to read as of with xCluster transactional replication in order to guarantee consistency and atomicity.)
```

Comment:
Can we just link to the architecture section for more details about xcluster_safe_time instead of explaining it here ?